docs/src/_reference/command-line-interface.md

```diff
@@ -9,6 +9,17 @@ Meltano provides a command line interface (CLI) that makes it easy to manage you
 To quickly find the `meltano` subcommand you're looking for, use the Table of Contents in the sidebar.
 For a better understanding of command line documentation syntax, the [docopt](http://docopt.org/) standard is useful.
 
+## Global Options
+
+The following options are available for all `meltano` subcommands:
+
+* `--log-config` - Path to a logging configuration file. See [Logging](/guide/logging) for more information.
+* `--log-level`: Set the log level for the command. Valid values are `debug`, `info`, `warning`, `error`, and `critical`.
+
+## Environment Variables
```

Comment:
The following configuration is available for all `meltano` subcommands via an environment variable: